lib/email/utils/helpers.js

```diff
@@ -39,13 +70,34 @@ function logFlowEventFromMessage(log, message, type) {
       flow_time: elapsedTime
     }
 
+    // If a locale exists and log it
```

Comment:
Just realized the parts to log sanitized locale in the translator do not land until Train-83, https://github.com/mozilla/fxa-auth-server/pull/1702. Since I wanted to do a point release on 82, going to hold off on logging locale till then.